clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/controllers/LoadBalancerController.groovy

```diff
@@ -26,19 +26,18 @@ import org.springframework.web.bind.annotation.RequestMethod
 import org.springframework.web.bind.annotation.RestController
 
 @RestController
-@RequestMapping("/applications/{application}/loadBalancers")
 class LoadBalancerController {
 
   @Autowired
   List<LoadBalancerProvider> loadBalancerProviders
 
   @PreAuthorize("hasPermission(#application, 'APPLICATION', 'READ')")
-  @RequestMapping(method = RequestMethod.GET)
+  @RequestMapping(value = "/applications/{application}/loadBalancers", method = RequestMethod.GET)
   Set<LoadBalancer> list(@PathVariable String application) {
-    ((List<LoadBalancer>) loadBalancerProviders.findResults {
+    loadBalancerProviders.findResults {
       it.getApplicationLoadBalancers(application)
-    })
-      .flatten()
-      .sort { a, b -> a.name.toLowerCase() <=> b.name.toLowerCase() }
+    }
+    .flatten()
+    .sort { a, b -> a.name.toLowerCase() <=> b.name.toLowerCase() } as Set<LoadBalancer>
```

Comment:
Not sure why it is returned as as a Set. I hope not for deduplication purposes, because I'm now making it return a List